src/periphery/FeeSplitter.sol

```diff
@@ -16,13 +20,52 @@ contract FeeSplitter is Ownable2Step {
   // Admin //
   ///////////
 
-  function setSplit(uint splitPercent) external onlyOwner {}
+  function setSplit(uint splitPercent) external onlyOwner {
+    require(_splitPercent <= 100, "Invalid percentage");
+    splitPercent = _splitPercent;
+  }
 
-  function setSubAccounts(uint accountA, uint accountB) external onlyOwner {}
+  function setSubAccounts(uint _accountA, uint _accountB) external onlyOwner {
+    accountA = _accountA;
+    accountB = _accountB;
+  }
 
   //////////////
   // External //
   //////////////
   /// @notice Work out the balance of the subaccount held by this contract, and split it based on the % split
-  function split() external {}
+  function split() external {
+    ISubAccounts.AssetBalance[] memory balances = subAccounts.getAccountBalances(subAcc);
+
+    for (uint i = 0; i < balances.length; i++) {
```

Comment:
I wouldn't expect any balances other than cash; and if somehow a different balance was added, its probably better that this just ignores them.

I'd specify cashAsset in constructor and query the balance directly